manticore/core/smtlib/constraints.py

```diff
@@ -101,12 +99,6 @@ def add(self, constraint, check=False):
 
         self._constraints.append(constraint)
 
-        if check:
-            from ...core.smtlib import solver
-
-            if not solver.check(self):
```

Comment:
Nobody was using it. "solver.check" does not exists. This should be Z3Solver.instance().check

Reply:
Whoops, in that case, I stand corrected. Nonetheless, if you've fixed it in another branch and are using it to debug, we should leave it in.